powerline/segments/i3wm.py

```diff
@@ -28,7 +29,8 @@ def workspaces(pl, only_show=None, output=None, strip=0):
 		are shown.
 
 	:param str output:
-		If specified, only workspaces on this output are shown.
+		If specified, only workspaces on this output are shown. If unspecified, 
+		may be set in ``segment_info``.
```

Comment:
`segment_info` is for developers. This documentation is for users. One should say or reference how this value may appear in `segment_info`, but *not* talk about `segment_info` itself.